common/src/service/QuerySpecification.cs

```diff
@@ -12,11 +12,5 @@ internal class QuerySpecification
     {
         [JsonProperty(PropertyName = "query", Required = Required.Always)]
         public string Sql { get; set; }
-
-        [JsonIgnore]
-        public int? PageSize { get; set; }
```

Comment:
I suspect this is another case of things a customer may reference, and we just don't have any tests for it. Do we know this is for sure dead code?

Reply:
Never mind; another internal class.